lib/views/pages/home/content/feed/components/comment_count.dart

```diff
@@ -16,16 +17,20 @@ class CommentCount extends ConsumerWidget {
   @override
   Widget build(BuildContext context, WidgetRef ref) {
     final asyncCount = ref.watch(postCommentCountProvider(postId));
-    final count = asyncCount.value ?? 0;
+    final countDetails = asyncCount.value ?? CommentCountDetails.empty;
 
-    const icon = Icon(Icons.comment, size: 20);
-    final countText = Text(count.toString());
+    final icon = Icon(
+      Icons.comment,
+      size: 20,
+      color: countDetails.isIconBlue ? Colors.blue : null,
+    );
+    final countText = Text(countDetails.count.toString());
```

Comment:
What do you think of displaying the number of comments in a user friendly way, like "99+" when reaching 100. Would also help avoid overflow.

Reply:
That's is actually a very good point !
Thanks for pointing it out :+1: 

Effectively, displaying the raw number of comments might not be very user friendly.
However, I believe that it might be a little outside the scope of this PR that focuses on the color of the icon and we might want to discuss how we display such high numbers of comments with the team. 
Thus, I believe that this might better be done in another PR if that's ok for you ?

Reply:
Yeah I agree, I thought about it too :)
Let's backlog it

Reply:
Will still have an overflow problem though.
Still not your PR, should we do a PR that fixes it by using "99+"?
![WhatsApp Image 2024-05-29 at 13 28 28_639beb9a](https://github.com/ProximaEPFL/proxima/assets/101193378/dad203ab-ab50-449a-a581-f59dfddc70f3)

Reply:
Good point thanks :+1: 

I have added the bug to the backlog on #360.

Reply:
In backlog: [#361](https://github.com/ProximaEPFL/proxima/issues/361)